products/rhel8/profiles/stig.profile

```diff
@@ -880,7 +880,7 @@ selections:
     - file_permissions_etc_audit_rulesd
 
     # RHEL-08-030620
-    - file_audit_tool_permissions
+    - file_audit_tools_permissions
```

Comment:
Please update the profile stability file as well.

Reply:
I always forget about this file.